src/spec.c

```diff
@@ -1657,6 +1659,19 @@ int bit(t_fieldMask id) {
   return 0;
 }
 
+// Return the current vesrion of the spec. 
+// The value of the version number does'nt indicate if the index
```

Comment:
doesn't

Also, what do you mean by "the index is newer or older?"

Reply:
since we increase the version by 1 each time we can get to an overflow and the count will start from 0, so not necessarily old_v_number < new_v_number